dev/com.ibm.ws.beanvalidation.v20_fat/fat/src/com/ibm/ws/bval/v20/fat/BeanVal20Test.java

```diff
@@ -30,18 +35,44 @@
 
     public static final String REG_APP = "bvalApp";
     public static final String CDI_APP = "bvalCDIApp";
+    public static final String MULTI_VAL_APP = "MultipleValidationXmlWeb";
 
     @Server("beanval.v20_fat")
     @TestServlets({
                     @TestServlet(servlet = BeanVal20TestServlet.class, contextRoot = REG_APP),
-                    @TestServlet(servlet = BeanValCDIServlet.class, contextRoot = CDI_APP)
+                    @TestServlet(servlet = BeanValCDIServlet.class, contextRoot = CDI_APP),
+                    @TestServlet(servlet = BeanValidationTestServlet.class, contextRoot = MULTI_VAL_APP)
     })
     public static LibertyServer server;
 
     @BeforeClass
     public static void setUp() throws Exception {
+
+        JavaArchive multiValXmlEjb1 = ShrinkHelper.buildJavaArchive("MultipleValidationXmlEjb1.jar", "bval.v20.ejb1.*");
```

Comment:
Building the 2 jars and 1 war can be simplified with `ShrinkHelper` static methods